docs/reference/api/docker_remote_api.md

```diff
@@ -64,12 +64,20 @@ without protocol. Throughout this structure, double quotes are required.
 
 ## Using Docker Machine with the API
 
-If you are using `docker-machine`, the Docker daemon is on a virtual host that uses an encrypted TCP socket. This means, for Docker Machine users, you need to add extra parameters to `curl` or `wget` when making test API requests, for example:
+If you are using `docker-machine`, the Docker daemon is on a virtual
+host that uses an encrypted TCP socket. This means, for Docker Machine
```

Comment:
I also think the text should've been "virtual machine" instead of "virtual host". Agreed on removing "virtual", or simply "the Docker daemon uses an encrypted .."